### PR TITLE
Image

### PR DESCRIPTION
**Pull request by chanchal-panpaliya**
project live - https://solutionground.netlify.app/

### CODE REVIEW

File: image/Image/Image.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    width: 50px;
+    height: 50px;
+}
+
+.image-size-medium{
+    width: 100px;
+    height: 100px;
+}
+
+.image-size-larger{
```
Comment (Mohasin-Haque) [inline image/Image/Image.css]: Please try to use relative units everywhere..If you have time .. else you can do it after 16

**Review by Mohasin-Haque — COMMENTED** [review]
You need to run prettier or format the code first. Then remove the commented part and also do care of using relative units. In every file. And remove the empty files

File: image/Image/Image.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    width: 12.5rem;
+    height: 12.5rem;
+}
+
+/*image responsive restricted upto 25rem*/
+.image-responsive{
+    width: 100%;
+    max-width: 25rem;
```
Comment (prathmeshjagtap) [inline image/Image/Image.css]: why the image was restricted to max-width: 25 rem,
Also, you can add object-fit: contain/cover to make it more precise.

**Review by prathmeshjagtap — CHANGES_REQUESTED** [review]
Code seems good you have used constant naming convention throughout your code so it becomes easy to understand.

**Review by prathmeshjagtap — APPROVED** [review]

**Review by SanjayJatti — APPROVED** [review]

**Comment by SanjayJatti** [comment]
Code is clean. You can go ahead.

**Pull request merged**